app/main/posts/modify/post-value-edit.html

```diff
@@ -201,49 +201,17 @@
           </label>
         </div>
       </div>
-
     <!-- type: categories -->
-      <div ng-switch-when="tags">
-        <div
-          class="form-field checkbox"
-          ng-repeat="option in attribute.options"
-          ng-if="option.parent_id === null"
-          >
-        <label>
-          <input
-                type="checkbox"
-                checklist-model="post.values[attribute.key]"
-                name="values_{{attribute.id}}_{{option.id}}"
-                checklist-value="option.id"
-                value="option.id"
-                ng-click="selectParent(option, attribute.key)"
-            >
-                {{option.tag}}
-        </label>
-        <div
-          class="form-field checkbox"
-          ng-if="option.children"
-          ng-repeat="child in option.children"
-        >
-          <label>
-          <input
-            type="checkbox"
-            checklist-model="post.values[attribute.key]"
-            name="values_{{attribute.id}}_{{child.id}}"
-            checklist-value="child.id"
-            value="child.id"
-            ng-click="selectChild(child, attribute.key)"
-          >
-            {{child.tag}}
-        </label>
-    </div>
-    </div>
-
-      <add-category
-        form-id="post.form.id"
-        attribute="attribute"
-        post-value="post.values[attribute.key]"
-      ></add-category>
+    <div ng-switch-when="tags">
+        <category-selector
+            available="attribute.options"
```

Comment:
Isn't this now only the ID of each category? not the full object? I don't see anywhere where you load the full tag inside category-selector... or did I miss it?

Reply:
its happening when loading the tags in the post-editor